src/query/core/buildInitiate.ts

```diff
@@ -87,10 +87,85 @@ export type MutationActionCreatorResult<
     >
   >
 > & {
-  arg: QueryArgFrom<D>
+  arg: {
+    /**
+     * The name of the given endpoint for the mutation
+     */
+    endpointName: string
+    /**
+     * The original arguments supplied to the mutation call
+     */
+    originalArgs: QueryArgFrom<D>
+    /**
+     * Whether the mutation is being tracked in the store.
+     */
+    track?: boolean
+    /**
+     * Timestamp for when the mutation was initiated
+     */
+    startedTimeStamp: number
+  }
+  /**
+   * A unique string generated for the request sequence
+   */
   requestId: string
+  /**
+   * A method to cancel the mutation promise. Note that this is not intended to prevent the mutation
+   * that was fired off from reaching the server, but only to assist in handling the response.
+   *
+   * Calling `abort()` prior to the promise resolving will make it throw with an error like:
```

Comment:
I actually have no idea what that would do, have you tested that? I assume it would resolve with `{ error: { name: 'AbortError', message: 'Aborted' }` unless you unwrapped it.

Reply:
Ah yeah my wording is definitely off. When I wrote it, in my head I was specifically thinking of the unwrapped promise, but that's not what I actually wrote.

I'll re-word it to make sense

Reply:
Re-worded to refer to 'error state' rather than saying it will 'throw'

Reply:
Also fwiw, it behaves exactly like you thought for the 'wrapped' promise